README.md

```diff
@@ -89,7 +89,7 @@ fn main() -> Result<(), Box<dyn Error>> {
         })
         .run(|model, fit, num| {
             println!("Generation: {} score: {:.3?}\t{:?}", num, fit, model.as_string());
-            fit == 12.0 || num == 500
+            (fit == 12.0).abs() < 0.1 || num == 500

```

Comment:
looks like a mistake (`abs()` of a bool)

Reply:
Yup that should be -.

Reply:
You're right, that sould be a - instead of ==